Makefile

```diff
@@ -58,10 +58,16 @@ test-integration:
 
```

Comment:
I was thinking of leaving the integration test tag because it seems like it could be useful. But, I will add a comment saying we don't currently use it.